django_scheduled_tasks/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from .models import ScheduledTask
+
+
+class ScheduledTaskAdmin(admin.ModelAdmin):
+    readonly_fields = ('last_timestamp', 'last_success')
+
+
+admin.site.register(ScheduledTask, ScheduledTaskAdmin)
```

Comment:
I've recently discovered a shorter way of registering ModelAdmmin classes that removes having to type the model admin class name twice:

```
@admin.register(models.XXX)
class YYY(admin.ModelAdmin):
    ...
```

Reply:
Fixed in 8ccf76a